docs/_files/input_parameters_list.csv

```diff
@@ -0,0 +1,3 @@
+,:Default:,:Definition:,:Example:,:Restrictions:,:Type:,:Unit:,label,ref,:Category:
```

Comment:
we only need to edit one source file for the inputs

Reply:
How do I unclude the csv under _files in the tool_interface.rst?